src/lib/config-db.ts

```diff
@@ -0,0 +1,66 @@
+import { Level } from 'level'
```

Comment:
Why is level needed here? 

Wouldn't it be possible to simplify this by using IndexedDB directly?

Reply:
fixed